### ISSUE TITLE
emojis not show

### ISSUE THREAD

**Issue by likezjuisee** [ManrajGrover/halo#25]
![image](https://user-images.githubusercontent.com/12730220/31419015-eea210c4-ae6b-11e7-8996-15f5fd191be6.png)

**Comment by ManrajGrover** [comment]
@likezjuisee I would need more information here. Please share the following:

1. Operating System?
2. Terminal being used?
3. Does your terminal support UTF-8?

**Comment by likezjuisee** [comment]
ubuntu
no terminal, I connected the system by the ssh client.

**Comment by ManrajGrover** [comment]
@likezjuisee Okay. You're using `ubuntu` on remote desktop? If yes, which operating system you are using to access this system. Also, which terminal is being used for the same?

If not, could you please check if your terminal supports UTF-8? You can do this by running `echo $LC_CTYPE` or `echo $LANG`.

**Comment by JungWinter** [comment]
@likezjuisee It is dependent on your ssh client program. Check whether supporting UTF-8 and Unicode charset or not. In my case, `PuTTY` client on `Windows 10` and `ubuntu 16.04`, it works well.

**Comment by likezjuisee** [comment]
thx

**Comment by ManrajGrover** [comment]
@likezjuisee Is this resolved?

**Comment by likezjuisee** [comment]
Sorry, I have not try the putty client yet. I will notice you when I have the result. Thanks.

**Comment by ManrajGrover** [comment]
@likezjuisee Was this issue resolved?

**Comment by norweeg** [comment]
@manrajgrover I have this issue across many terminals as well as in Jupyter notebooks.  I can only ever get the "line" spinner, no matter what value I put

**Comment by manrajgrover** [comment]
@norweeg Are you on Windows system?

**Comment by norweeg** [comment]
@manrajgrover yep, already found the line in your code.  I'll try to fix it for you.  I get that terminal support sucks, but you shouldn't be preventing it in Jupyter on Windows....

**Comment by norweeg** [comment]
This isn't so much a bug as it is an unnecessary limitation placed on Windows users.  If you remove this limit and use a terminal emulator that supports Unicode output on Windows, then you don't have this problem.  See my comment here: https://github.com/manrajgrover/halo/issues/5#issuecomment-596145486
